worlds/sc2/docs/en_Starcraft 2.md

```diff
@@ -18,50 +20,81 @@ You find items by making progress in these categories:
 
-The goal is to beat the final mission in the mission order. The yaml configuration file controls the mission order and how missions are shuffled.
+The goal is to beat the final mission in the mission order. 
+The yaml configuration file controls the mission order (e.g. blitz, grid, etc.), which of the four Starcraft 2 
+campaigns can be used to populate the mission order and how missions are shuffled. 
+Since the first two options determine the number of missions in a Starcraft 2 world, they can be used to customize the 
+expected time to complete the world. 
+Note that the evolution missions from Heart of the Swarm are not included in the randomizer.
 
 ## What non-randomized changes are there from vanilla Starcraft 2?
 
 1. Some missions have more vespene geysers available to allow a wider variety of units.
-2. Many new units and upgrades have been added as items, coming from co-op, melee, later campaigns, later expansions, brood war, and original ideas.
-3. Higher-tech production structures, including Factories, Starports, Robotics Facilities, and Stargates, no longer have tech requirements.
+2. Many new units and upgrades have been added as items, coming from co-op, melee, later campaigns, later expansions, 
+brood war, and original ideas.
+3. Higher-tech production structures, including Factories, Starports, Robotics Facilities, and Stargates, no longer 
+have tech requirements.
 4. Zerg missions have been adjusted to give the player a starting Lair where they would only have Hatcheries.
-5. Upgrades with a downside have had the downside removed, such as automated refineries costing more or tech reactors taking longer to build.
-6. Unit collision within the vents in Enemy Within has been adjusted to allow larger units to travel through them without getting stuck in odd places.
+5. Upgrades with a downside have had the downside removed, such as automated refineries costing more or tech reactors 
+taking longer to build.
+6. Unit collision within the vents in Enemy Within has been adjusted to allow larger units to travel through them 
+without getting stuck in odd places.
 7. Several vanilla bugs have been fixed.
 
 ## Which of my items can be in another player's world?
 
-By default, any of StarCraft 2's items (specified above) can be in another player's world. See the
-[Advanced YAML Guide](/tutorial/Archipelago/advanced_settings/en)
-for more information on how to change this.
+By default, any of StarCraft 2's items (specified above) can be in another player's world. 
+See the [Advanced YAML Guide](/tutorial/Archipelago/advanced_settings/en) for more information on how to change this.
 
 ## Unique Local Commands
 
-The following commands are only available when using the Starcraft 2 Client to play with Archipelago. You can list them any time in the client with `/help`.
+The following commands are only available when using the Starcraft 2 Client to play with Archipelago. 
+You can list them any time in the client with `/help`.
```

Comment:
Is it worth mentioning that `/received` has custom behaviour through the sc2 client? The command is available through common client, but won't organize items based on faction/unit effects, and doesn't have filtering by faction, name, or unit group.

Reply:
I had no idea that '/received' is now different with SC2 client. Can you provide a text sample that describe the feature? If memory serves right, basic '/received' is ordered by the order of reception. Thus that mean that  '/received' in sc2 tab is by default organized in " faction/unit effects" and, when provided with an argument, only shows item received in the provided faction/name/unitGroup ?

Reply:
`/received` works has custom behaviour in the Starcraft 2 client. Instead of ordering strictly by order received, items are grouped by their parent items, which are items that are the sole method of utilizing an upgrade. Items with multiple parents, such as Void Ray class-wide upgrades, are put in the top-level category. An optional filter parameter may be passed to `/received` to filter by race, item group, or just include items containing the parameter as a search term.

### Examples (for your reference)
![image](https://github.com/user-attachments/assets/cf5f7c7c-bcff-45a4-8edd-ac6002fe8ce2)
![image](https://github.com/user-attachments/assets/841fe7af-ae00-4b7f-a6c9-66d346ee1451)
![image](https://github.com/user-attachments/assets/734470c1-4baf-4fac-bf40-69522e77169a)
![image](https://github.com/user-attachments/assets/6c6e1ea7-66a9-40d1-8be5-8167484fbccc)

Reply:
Is there a logic in the order of the items when shown inside a group/nestedGroup?

Reply:
I added a paragraph about that in the docs, both in EN and FR. (Not that you need to check it, just stating it in case someone see the discussion and wonder if it was done or not.)

Reply:
> Is there an order [...]
The order appears to be by item ID from what I can tell, so there's no information about order received.